.gitignore

```diff
@@ -1,6 +1,7 @@
 /node_modules
 /coverage
 /dist
+/.vscode
```

Comment:
Can you keep this out of the repository, instead of having to add your IDE of choice's configuration directory to each project you work on it's usually easier to set up a global gitignore: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/